proc_IR.py

```diff
@@ -11,27 +11,35 @@
 clock_correction = 1.785
 filter_bandwidth = 5e3
 coh_sel = {'ph1':0,
-        'ph2':1}
+        'ph2':0}
```

Comment:
doesn't make sense

Reply:
shouldn't it be Δp=-1??

Reply:
So there was coherence selection in the preprocessing and this messed up a bit of things such as the hermitian function later in the code so I did some rearranging. There was some trouble running the fitting but I fixed and it works on my end thought the guess is completely wrong but aside from the guess it's good!

Reply:
for an inversion recovery, the coherence selection should be ph1=0, ph2=-1.  Can't mere until that's fixed. This is a conceptual error, whether or not you coincidentally get the right result.

Reply:
I understand this is what we should see however, in the frequency domain prior to coherence selection this is the figure
![image](https://user-images.githubusercontent.com/52217688/93219686-9d270c80-f739-11ea-83c0-fd3527b8ea3a.png)
Totally not sure if you are able to see that.... however in short the signal is in ph2=0 ph1=0... which is very odd

Reply:
this is bad data